paasta_itests/steps/setup_marathon_job_steps.py

```diff
@@ -110,3 +123,33 @@ def can_run_get_app(context):
 @then(u'we should see the number of instances become {number:d}')
 def assert_instances_equals(context, number):
     assert context.marathon_client.get_app(context.app_id).instances == number
+
+
+@when(u'we mark the host "{host}" as at-risk')
+def mark_host_at_risk(context, host):
+    start = paasta_maintenance.datetime_to_nanoseconds(paasta_maintenance.now())
+    duration = paasta_maintenance.parse_timedelta('1h')
+    config = {
+        'master': '%s' % get_service_connection_string('mesosmaster'),
+        'scheme': 'http',
+        'response_timeout': 5,
+    }
+    with contextlib.nested(
+        mock.patch('paasta_tools.paasta_maintenance.load_credentials', autospec=True),
+        mock.patch.object(mesos.cli.master, 'CFG', config),
+    ) as (
+        mock_load_credentials,
+        _,
+    ):
+        mock_load_credentials.side_effect = paasta_maintenance.load_credentials(mesos_secrets='/etc/mesos-slave-secret')
+        paasta_maintenance.drain([host], start, duration)
+
+
+@then(u'the tasks on the host "{host}" should be drained')
```

Comment:
Good point. How about we make it a bit more flexible and go with, `there should be {number:d} tasks on the host "{host}"`?